html/donate/en.html

```diff
@@ -9,10 +9,10 @@
 
     <meta property="fb:app_id" content="219331381518041" />
     <meta property="og:type" content="article" />
-    <meta property="og:title" content="Help us fund the Open Food Facts 2024 budget!" />
+    <meta property="og:title" content="Help us lead the change in the food sector!" />
     <meta
       property="og:url"
-      content="https://world.openfoodfacts.org/donate-to-openfoodfacts"
+      content="https://world.openfoodfacts.org/donate-to-open-food-facts"
```

Comment:
@stephanegigandet we should put an UTM in there 
```suggestion
      content="https://world.openfoodfacts.org/donate-to-open-food-facts?utm_source=donate-page&utm_campaign=donation-2024&utm_content=donate-page-en"
```

Reply:
OK but adding utm tags to an open graph urls is the kind of things that is likely to have unforeseen side effects.